spec/spec_helper.rb

```diff
@@ -53,6 +53,11 @@
     add_group 'VBADocuments', 'modules/vba_documents/'
     add_group 'Veteran', 'modules/veteran/'
     add_group 'VeteranVerification', 'modules/veteran_verification/'
+    add_group 'OpenidAuth', 'modules/openid_auth/'
```

Comment:
This is the only reference I can see in the PR to `OpenidAuth` ->  how is this change related? 

Can we do it in a separate commit/pr?